dissect/target/plugins/os/windows/ual.py

```diff
@@ -192,7 +195,7 @@ def populate_role_guid_map(self):
                 "role_name": record.get("RoleName"),
             }
 
-    def read_table_records(self, table_name):
+    def read_table_records(self, table_name) -> Iterator[tuple]:
```

Comment:
```suggestion
    def read_table_records(self, table_name: str) -> Iterator[tuple]:
```
I also see a blasphemous f-string log in this function, can you remove that? Thanks :smile: